gettsim/social_insurance_contributions/ges_krankenv.py

```diff
@@ -220,12 +536,48 @@ def _ges_krankenv_beitr_midi_job_m(
     -------
 
     """
-    gesamtbeitrag_midi_job_m = (
-        soz_vers_beitr_params["beitr_satz"]["ges_krankenv"]["an"]
-        + soz_vers_beitr_params["beitr_satz"]["ges_krankenv"]["ag"]
-    ) * midi_job_bemessungsentgelt_m
+    if in_gleitzone:
+        out = (
+            bruttolohn_m
+            * soz_vers_beitr_params["beitr_satz"]["ges_krankenv"]["allgemein"]
+            / 2
+        )
+    else:
+        out = 0.0
 
-    ag_beitr_midi_job_m = (
-        bruttolohn_m * soz_vers_beitr_params["beitr_satz"]["ges_krankenv"]["ag"]
-    )
-    return gesamtbeitrag_midi_job_m - ag_beitr_midi_job_m
+    return out
+
+
+def an_beitr_ges_krankenv_midi_job(
+    ges_krankenv_beitr_midi_job_m: float,
+    ag_beitr_ges_krankenv_midi_job: float,
+) -> float:
+    """Calculating the employer health insurance contribution.
+
+
+    Parameters
+    ----------
+    ges_krankenv_beitr_midi_job_m
+        See :func:`ges_krankenv_beitr_midi_job_m`.
+    ag_beitr_ges_krankenv_midi_job
+        See :func:`ag_beitr_ges_krankenv_midi_job`.
+    Returns
+    -------
+
+    """
+    return ges_krankenv_beitr_midi_job_m - ag_beitr_ges_krankenv_midi_job
+
+
+def selbstständig_ges_krankenv(selbstständig: bool, in_priv_krankenv: bool) -> bool:
```

Comment:
Is this used anywhere?

Reply:
Probably a reminiscent of what is `ges_krankenv_beitr_selbst_m()` by now.